tests/textcase/plugin/options/substitue_command_name_spec.lua

```diff
@@ -0,0 +1,20 @@
+local textcase = require("textcase")
+local test_helpers = require("tests.test_helpers")
+
+describe("plugin options substitude_command_name", function()
+  before_each(function()
+    textcase.setup({
+      substitude_command_name = "Magic",
+    })
+
+    local buf = vim.api.nvim_create_buf(false, true)
+    vim.api.nvim_command("buffer " .. buf)
+    vim.api.nvim_buf_set_lines(0, 0, -1, true, { "LoremIpsum", "LoremNunc" })
+  end)
+
+  it("should execute the substitude command with the passed in name", function()
```

Comment:
Shall we also add a test for the default value?

Reply:
Good idea 👍 done